qiskit/qobj/pulse_qobj.py

```diff
@@ -76,6 +76,10 @@ def __eq__(self, other):
 class PulseQobjInstruction:
     """A class representing a single instruction in an PulseQobj Experiment."""
 
+    _SIMPLE_ATTRS = ['ch', 'conditional', 'val', 'phase', 'frequency',
```

Comment:
Maybe
```suggestion
    _COMMON_ATTRS = ['ch', 'conditional', 'val', 'phase', 'frequency',
```
or 
```suggestion
    _KNOWN_ATTRS = ['ch', 'conditional', 'val', 'phase', 'frequency',
```

Reply:
Common sounds good enough to me. Thanks for the wording suggestion :)